### PR TITLE
Cleanup flex

### PR DESCRIPTION
**Pull request by msukkar**
@aw3as

### CODE REVIEW

**Comment by keenadams** [comment]
Username should probably stay lowercased.

**Comment by msukkar** [comment]
Everything else on the navbar was uppercase. Wasn't sure if deliberate (since username is the only non-static value). Can change it back though.

File: app/assets/javascripts/courses.js [hunk 1]
```diff
@@ -131,11 +130,12 @@ ready = function() {
...
-			success: function(response) {
-				alert('Course saved for scheduler!');
-			},
-			failure: function(response) {
-				alert('Could not load corresponding course!');
+			success: function (data) {
+				if (data.saved === "True") {
+					$('#save-course').html('<div class=\"btn btn-primary\" id=\"save-course\"> Unsave Course </div>');
+				} else {
+					$('#save-course').html('<div class="btn btn-primary" id="save-course"> Save Course </div>');
```
Comment (msukkar) [inline app/assets/javascripts/courses.js]: Probably a better way of doing this. Was best I could come up with no wifi

Comment (keenadams) [inline app/assets/javascripts/courses.js]: lol I actually already did this.

**Pull request merged**